harvester/dmat/src/main/java/dk/dbc/dataio/harvester/dmat/ScheduledHarvestBean.java

```diff
@@ -0,0 +1,65 @@
+package dk.dbc.dataio.harvester.dmat;
+
+import dk.dbc.dataio.harvester.AbstractScheduledHarvestBean;
+import dk.dbc.dataio.harvester.types.DMatHarvesterConfig;
+import dk.dbc.util.RunSchedule;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ejb.EJB;
+import javax.ejb.ScheduleExpression;
+import javax.ejb.Singleton;
+import javax.ejb.Startup;
+import java.time.ZoneId;
+import java.util.Date;
+
+/**
+ * This singleton Enterprise Java Bean (EJB) class schedules harvests every 60 seconds
+ */
+@Singleton
+@Startup
+public class ScheduledHarvestBean extends AbstractScheduledHarvestBean<HarvesterBean,
+        DMatHarvesterConfig, HarvesterConfigurationBean> {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ScheduledHarvestBean.class);
+
+    @EJB HarvesterBean harvesterBean;
+    @EJB HarvesterConfigurationBean harvesterConfigurationBean;
+
+    @Override
+    public HarvesterBean getHarvesterBeanImpl() {
+        return harvesterBean;
+    }
+
+    @Override
+    public HarvesterConfigurationBean getHarvesterConfigurationBeanImpl() {
+        return harvesterConfigurationBean;
+    }
+
+    @Override
+    public ScheduleExpression getTimerSchedule() {
+        final ScheduleExpression scheduleExpression = new ScheduleExpression();
+        scheduleExpression.second("0");
+        scheduleExpression.minute("*");
+        scheduleExpression.hour("*");
+        return scheduleExpression;
+    }
+
+    @Override
+    public Logger getLogger() {
+        return LOGGER;
+    }
+
+    @Override
+    public boolean canRun(DMatHarvesterConfig config) {
+        try {
+            final RunSchedule runSchedule = new RunSchedule(config.getContent().getSchedule())
+                    .withTimezone(ZoneId.of(System.getenv("TZ")));
```

Comment:
Oppe i harvestoperation laver du gymnastik hvis getenv returnerer null, men ikke her - what's the difference ?

Reply:
Her er timezone required og vi ønsker en hård fejl fra pod'en hvis den ikke er defineret. 
Vi kan leve med manglende timezone i forhold til addimetadata i harvestoperation.